win/main.go

```diff
@@ -49,20 +53,38 @@ func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	cmd := runServer(ctx)
 
-	ui, err := lorca.New("http://localhost:8088", "", 1024, 768)
+	defer func() {
+		log.Println("Cleanup")
+		cancel()
+		if err := cmd.Process.Kill(); err != nil {
+			log.Fatal("failed to kill server process: ", err)
+		}
+
+		if r := recover(); r != nil {
```

Comment:
This new recover here is capturing the panics at lorca.New and ui.Load? for example? makes sense?

Reply:
tbh, I'm not sure what the best pattern here is. The original issue was cleanup wasn't being called on crashes, so I deferred it. In the cleanup, the app is exiting anyway, so I added the recover and log to hide the stack trace. What would you recommend instead?